src/jimgw/core/single_event/likelihood.py

```
@@ -399,7 +400,7 @@ def _init_time_marginalization(self, tc_range: tuple[Float, Float]) -> None:
         )
         self.pad_high = jnp.zeros(max(0, n_pad_high))
 
-    def _reduce_time(self, complex_d_inner_h: Float[Array, " n_freq"]) -> Float:
+    def _reduce_time(self, complex_d_inner_h: Float[Array, " n_freq"]) -> FloatScalar:
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)